ec3730/Views/Interface/InterfaceView.swift

```diff
@@ -71,6 +71,7 @@ struct InterfaceView: View {
         CopyCellView(title: "Runing", content: interface.isRunning ? "Yes" : "No")
         CopyCellView(title: "Up", content: interface.isUp ? "Yes" : "No")
         CopyCellView(title: "Supports Multicast", content: interface.supportsMulticast ? "Yes" : "No")
+        CopyCellView(title: "Data Usage", rows: [CopyCellRow(title: "Send / Recieve", content: interface.dataUsage)])
```

Comment:
If this is consistently zero we might as well not display this information

Reply:
I agree